src/tests/ftest/util/data_mover_test_base.py

```diff
@@ -616,214 +614,118 @@ def dataset_verify(self, obj_list, cont, num_objs, num_dkeys,
             obj.close()
         cont.close()
 
-    def set_datamover_params(self,
-                             src_type=None, src_path=None,
-                             src_pool=None, src_cont=None,
-                             dst_type=None, dst_path=None,
-                             dst_pool=None, dst_cont=None):
+    def set_dm_params(src_path=None, dst_path=None, dst_pool=None):
         """Set the params for self.tool.
-        Called by run_datamover if params are passed.
 
         Args:
-            src_type (str): how to interpret the src params.
-                Must be in PARAM_TYPES.
-            src_path (str): source cont path or posix path.
-            src_pool (TestPool, optional): the source pool or uuid.
-            src_cont (TestContainer, optional): the source cont or uuid.
-            dst_type (str): how to interpret the dst params.
-                Must be in PARAM_TYPES.
-            dst_path (str): destination cont path or posix path.
-            dst_pool (TestPool, optional): the destination pool or uuid.
-            dst_cont (TestContainer, optional): the destination cont or uuid.
-
+            src_path (str, optional): src daos://pool/cont/[path] or POSIX path.
+                Default is None, which is ignored.
+            dst_path (str, optional): dst daos://pool/cont/[path] or POSIX path
+                Default is None, which is ignored.
+            dst_pool (TestPool/str, optional): dst pool or uuid. Only used with serialization.
+                Default is None, which is ignored.
         """
         if self.tool == "DCP":
-            self._set_dcp_params(src_type, src_path, src_pool, src_cont,
-                                 dst_type, dst_path, dst_pool, dst_cont)
+            self._set_dcp_params(src_path, dst_path)
         elif self.tool == "DSYNC":
-            self._set_dsync_params(src_type, src_path, src_pool, src_cont,
-                                   dst_type, dst_path, dst_pool, dst_cont)
+            self._set_dsync_params(src_path, dst_path)
         elif self.tool == "DSERIAL":
-            assert src_type in (None, "DAOS", "DAOS_UUID") #nosec
-            assert src_path is None #nosec
-            assert dst_type in (None, "DAOS", "DAOS_UUID") #nosec
-            assert dst_path is None #nosec
-            assert dst_cont is None #nosec
-            self._set_dserial_params(src_pool, src_cont, dst_pool)
+            self._set_dserial_params(src_path, dst_pool)
         elif self.tool == "FS_COPY":
-            self._set_fs_copy_params(src_type, src_path, src_pool, src_cont,
-                                     dst_type, dst_path, dst_pool, dst_cont)
+            self._set_fs_copy_params(src_path, dst_path)
```

Comment:
(pylint-protected-access) Access to a protected member _set_fs_copy_params of a client class
(pylint-undefined-variable) Undefined variable 'self'